backend/api/Database/Models/Pose.cs

```diff
@@ -8,6 +8,7 @@ public class Orientation
     {
         [Key]
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        [Required]
```

Comment:
The `Key` attribute will make the property be non-null in the database always, adding the `required` attribute does not change anything database-wise, but complicates our code because it forces us to specify separate `query models` to avoid errors when receiving models without explicit ID's through the API.
Unless there was a specific other reason for adding this attribute, I suggest removing it and thus removing the need for more "duplicate" model classes.
(Same for the other model classes)

Reply:
I'm not certain this would remove the need for the query models we are currently using. If we don't use those models the user friendliness of the endpoints will be worse as you'll have to remove multiple sections. For example ID and some others for specific models.

As for the database issue me and @Christdej had an issue which we solved by setting required as the database was throwing an error specifically linked to that property in that model. I can test again to see if the issue is still there.

Reply:
More specifically the problem was related to the automatically generated first migration not working when setting up the database. If I remove the required option now it works fine but I'm not certain it would work if we built the database from scratch again. Not entirely certain what to do about it to be honest.

Reply:
Agreed to leave it for now and maybe get back to it in the future 👍